src/common/lib/mnemonic.ts

```diff
@@ -2,10 +2,15 @@ import * as secp256k1 from "@noble/secp256k1";
 import { HDKey } from "@scure/bip32";
 import * as bip39 from "@scure/bip39";
 
+export const LIQUID_DERIVATION_PATH = "m/44'/1776'/0'/0/0";
```

Comment:
We should consider adding a _REGTEST version too which will be controlled by the network setting.